curs2java/src/persons/BubbleSort.java

```diff
@@ -0,0 +1,25 @@
+package persons;
+import persons.Person;
+
+public class BubbleSort { /* Metoda BubbleSort primeste o lista de persoane si returneaza tot aceeasi lista, doar ca sortata
+*/
+	public static Person[] persoane (Person persoane[]) {
```

Comment:
Method and var names should be in english.
The _persoane_ word does not specify what the method does. It does sorting, why isn't it called that?